scripts/kind-e2e/lib_operator_deploy_subm.sh

```diff
@@ -21,65 +21,6 @@ function add_subm_gateway_label() {
   kubectl label node $context-worker "submariner.io/gateway=true" --overwrite
 }
 
```

Comment:
We still need the creation functions here; without them, `make e2e` fails:

    [scripts]$ create_subm_endpoints_crd
    ./../scripts/kind-e2e/e2e.sh: line 405: create_subm_endpoints_crd: command not found

If we always run the generation as part of e2e, or we ship the generated CRDs, then the creation function can be simplified to only refer to the already-available CRD file.